com.unity.ml-agents.extensions/Runtime/Sensors/ArticulationBodyPoseExtractor.cs

```diff
@@ -54,17 +54,17 @@ public ArticulationBodyPoseExtractor(ArticulationBody rootBody)
                 parentIndices[i] = bodyToIndex[parentArticBody];
             }
 
-            SetParentIndices(parentIndices);
+            Setup(parentIndices);
         }
 
         /// <inheritdoc/>
-        protected override Vector3 GetLinearVelocityAt(int index)
+        protected internal override Vector3 GetLinearVelocityAt(int index)
```

Comment:
why the internal addition?

Reply:
Wanted to be able to assert on it
https://github.com/Unity-Technologies/ml-agents/blob/7e75f129f15ce8288a09a25ebdcd8f88695dfc21/com.unity.ml-agents.extensions/Tests/Editor/Sensors/RigidBodyPoseExtractorTests.cs#L67